bluebottle/time_based/tests/test_admin.py

```diff
@@ -73,9 +73,42 @@ def test_list_activities_office(self):
         response = self.app.get(url)
         self.assertEqual(len(response.html.find_all("a", string="Schin op Geul")), 2)
 
+    def test_participant_count(self):
+        activity = DateActivityFactory.create()
+        DateParticipantFactory.create_batch(5, activity=activity)
+        url = reverse('admin:time_based_dateactivity_changelist')
+        response = self.app.get(url)
+        self.assertEqual(int(response.html.find_all('td')[5].get_text()), 5)
 
-class DateActivityAdminScenarioTestCase(BluebottleAdminTestCase):
+    def test_start_date(self):
+        activity = DateActivityFactory.create()
+        time_1 = now()
+        time_2 = now() - timedelta(days=2)
+        time_3 = now() + timedelta(days=3)
 
+        DateActivitySlotFactory.create(
+            activity=activity,
+            start=time_1
+        )
+        DateActivitySlotFactory.create(
+            activity=activity,
+            start=time_2
+        )
+        DateActivitySlotFactory.create(
+            activity=activity,
+            start=time_3
+        )
+
+        start_time = activity.slots.all()[0].start.astimezone(get_current_timezone())
+        self.assertTrue(start_time == time_2)
```

Comment:
This would be nicer
`self.assertEqual(start_time, time_2)`

Reply:
True, changed it 👌🏽